doc/Quick-Start-Guide.rst

```diff
@@ -24,10 +26,15 @@ Build
     cd cortx-motr
     sudo scripts/install-build-deps
 
-2. If using lnet as the transport, check the Lustre network interface configuration::
+2. If using lnet as the transport (default configuration),
```

Comment:
It would be better if we can move 2) and 3) points to the end of this doc.

Reply:
Why? These network configuration considerations are part of preparing the dependencies. Build options depend on what network the user chooses.

Reply:
Why? These network configuration considerations are part of preparing the dependencies. Build options depend on what network the user chooses.